src/test/java/org/gridsuite/cgmes/assembling/job/ProfilesAcquisitionJobTest.java

```diff
@@ -408,4 +407,30 @@ public void testAcceptedFilesConnection() throws IOException {
             e.printStackTrace();
         }
     }
+
+    public void truncate() {
+        JdbcConnector connector = cgmesAssemblingLogger.getConnector();
```

Comment:
I think we should simplify

truncate handled_files_dependencies;
truncate handled_files;
truncate import_files;